src/main/java/core/org/akaza/openclinica/dao/EventCRFSDVFilter.java

```diff
@@ -56,11 +56,18 @@ private String buildCriteria(String criteria, String property, Object value) {
                 String sdvStatusString = value.toString().trim();
                 if(sdvStatusString.contains("+")){
                     for(String sdvStatus: sdvStatusString.split("\\+")){
-                        sdvStatusFilterArray.add(SdvStatus.getByI18nDescription(sdvStatus.trim()).toString());
+                        if(sdvStatus == null || sdvStatus.toLowerCase().contains("null"))
+                            sdvStatusFilterArray.add(SdvStatus.NOT_VERIFIED.toString());
+                        else
+                            sdvStatusFilterArray.add(SdvStatus.getByI18nDescription(sdvStatus.trim()).toString());
                     }
                 }
-                else
-                    sdvStatusFilterArray.add(SdvStatus.getByI18nDescription(sdvStatusString).toString());
+                else{
+                    if(sdvStatusString == null || sdvStatusString.toLowerCase().contains("null"))
+                        sdvStatusFilterArray.add(SdvStatus.NOT_VERIFIED.toString());
```

Comment:
In the if condition above, you are already accessing "sdvStatusString" so if you need to add a null check for this, you should add it to the if condition above